docs/howto/how-to-make-tensorflow-plugin-with-ui/tensorflow-plugin-with-ui.md

```diff
@@ -0,0 +1,1488 @@
+        {modelTabs[selectedTab]}
+      </Paper>
+    </Container>
+  );
+}
+
+export default ModelSection;
+```
+
+#### Component(s): `./src/components/charting/line-plots.tsx`
+
+
+```jsx
+import React, {useState} from "react";
+
+import { AppBar, Container, useTheme } from '@alteryx/ui';
+
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+export const InteractiveLinePlot = ({labels, plotTitle, datasets, ...rest}) => {
+    const theme = useTheme();
+    let options = {
+  responsive: true,
+  plugins: {
+    legend: {
+      position: 'top' as const,
+    },
+    title: {
+      display: true,
+      text: plotTitle,
+        },
+    },
+    };
+
+    const data = {
+        labels,
+        datasets,
+        rest
+    }
+
+    return (
+    <Container>
+        <AppBar color="default" elevation={0} position="static">
+            {plotTitle} <br />
+            <Line options={options} data={data}  />;
+        </AppBar>
+        
+    </Container>
+  );
+}
+```
+
+## Testing End to End
+Now, you should see a GUI that looks like the below (E-Note: will add screencap doing one more fresh build for screens since updates):
+
+
+#### Figure: Tool GUI
+
+[figure here]
+
+
+## Package into a YXI
+
+First, we should ensure we have all our dependencies for the front and backend. Then, we run `create-yxi` to package each of these into a deployable using their respective package managers.
+
+### Python Dependencies
+
+Add `tensorflow==2.12.0` to your tool's `requirements-third-party.txt`.
+
+### React dependencies
+
+Run `npm install --save lodash chart.js`
+
+### CLI packaging command
+
+Run the `ayx_plugin_cli create-yxi` command, which bundles all the plugins in the workspace into a `.yxi` archive. It should look something like this:
+
+```powershell
+$ create-yxi
+[Creating YXI] started
+[Creating YXI] -- generate_config_files:generate_config_xml
+// output truncated //
+[Creating YXI] .  create_yxi:create_yxi
+[Creating YXI] finished
+```
+
+The resulting `.yxi` archive should exist in your plugin workspace's build/exit directory.
+
+## Install and Run in Designer
+
+### Method 1
+After you create a `.yxi,` you can double-click the `.yxi` or drag and drop it into Designer to install it. It then prompts you to install the package in a new dialog box. Which looks something like this:
+
+![YXI Install Dialog](./assets/install-yxi-dialog.jpg)
+
+Once it installs, you can find the plugin under the `Tensorflow Python SDK Examples` tool category.
+
+### Method 2
+You can also create the `.yxi` _**and**_ install it all in one step via the `designer-install` command. Choose the install option that matches your Designer install. Typically, this is the `user` install option. 
```

Comment:
TW review:

You can also create the `.yxi` _and_ install it all in one step via the `designer-install` command. Choose the install option that matches your Designer install. Typically, this is the `user` install option.